src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingView.macios.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System.Collections.ObjectModel;
+using System.Runtime.InteropServices;
+using CommunityToolkit.Maui.Core.Extensions;
+using CoreGraphics;
+using Foundation;
+using Microsoft.Maui.Platform;
+using UIKit;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+public partial class MauiDrawingView : UIView
+{
+	readonly UIBezierPath currentPath = new();
+	readonly List<UIScrollView> scrollViewParents = new();
+
+	CGPoint previousPoint;
+	MauiDrawingLine? currentLine;
+
+	/// <summary>
+	/// Line color
+	/// </summary>
+	public UIColor LineColor { get; set; } = Colors.Black.ToPlatform();
```

Comment:
Can we use the platform black color? This will avoid us calling some methods (`Get` method and `ToPlatform` methods)

Reply:
I just like the new Colors class and ToPlatform method 😄